### PR TITLE
Add templates

### PR DESCRIPTION
**Pull request by albertteoh**
Signed-off-by: albertteoh <albert.teoh@logz.io>

Relates to: https://github.com/jaegertracing/jaeger/issues/2974 

Copies the following files from https://github.com/jaegertracing/jaeger to here, unchanged:

- .github/ISSUE_TEMPLATE/bug_report.md
- .github/ISSUE_TEMPLATE/feature_request.md
- .github/ISSUE_TEMPLATE/config.yml (so we have one place for discussions)
- .github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question via Jaeger Community Support (online chat)
+    url: https://cloud-native.slack.com/archives/CGG7NFUJ3
```
Comment (yurishkuro) [inline .github/ISSUE_TEMPLATE/config.yml]: Some people get confused by you can't create an account at this link. We either need to include the other link that creates invites or point to a page that has both (like on the website)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Welcome to the Jaeger project! 👋🎉
```
Comment (yurishkuro) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]:
```suggestion
Please delete this comment before posting.

Welcome to the Jaeger project! 👋🎉
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!--
+We appreciate your contribution to the Jaeger project! 👋🎉
```
Comment (yurishkuro) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
Please delete this comment before posting.

We appreciate your contribution to the Jaeger project! 👋🎉
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+- You have read the guide for contributing
+  - See https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING.md
+- You signed all your commits (otherwise we won't be able to merge the PR)
+  - See https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md#certificate-of-origin---sign-your-work
+- You added unit tests for the new functionality
+- You mention in the PR description which issue it is addressing, e.g. "Resolves #123"
+-->
+
+## Which problem is this PR solving?
+-
```
Comment (yurishkuro) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- <!-- e.g. Resolves #123 -->
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question via Jaeger Community Support (online chat)
+    url: https://cloud-native.slack.com/archives/CGG7NFUJ3.
+    about: Please ask and answer questions here for faster response. If you don't have an account, signup at: https://slack.com/intl/en-au/get-started.
```
Comment (yurishkuro) [inline .github/ISSUE_TEMPLATE/config.yml]: The sign up url is https://slack.cncf.is

Comment (albertteoh) [reply .github/ISSUE_TEMPLATE/config.yml]: Thanks, updated the URL.

**Review by yurishkuro — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
```
Comment (yurishkuro) [inline .github/ISSUE_TEMPLATE/config.yml:2]: these don't seem to work in this repo (after merge)

Comment (albertteoh) [reply .github/ISSUE_TEMPLATE/config.yml:2]: indeed, thanks for checking. Addressed in: #2

Comment (yurishkuro) [reply .github/ISSUE_TEMPLATE/config.yml:2]: +1, works now: https://github.com/jaegertracing/jaeger-ui/issues/new/choose

**Pull request merged**